src/components/newsItem.js

```diff
@@ -11,13 +13,18 @@ export default function NewsItem({
   hoverEffects,
 }) {
   const renderNewsItem = () => {
+    const slug = convertTitleToSlug(title)
+
     return (
       // Later on we want to use something other than the contentful ID for the link
       <div
         className={hoverEffects ? "news-item news-item--hover" : "news-item"}
       >
         <div className="news-item__back-panel"></div>
-        <Link className="news-item__link" to={`/news/${id}`}>
+        <Link
+          className="news-item__link"
+          to={`/news${date ? "/" + date : ""}/${slug ? slug : id}`}
```

Comment:
Can we change `id` to `contenful_id` just to be consistent with the other places this is used?

Also, might be worth making an update in the future to create a custom Link component that can format these paths. Since the string is a bit complicated, it would be nice to avoid having to update it in multiple locations if possible.

Nothing we need to do now, but might be worth noting.